backend/repository/billings/repository.go

```diff
@@ -73,3 +73,31 @@ func (b *billingRepository) CreateBilling(ctx context.Context, billingDetail ent
 
 	return &billingDetail, nil
 }
+
+func (b *billingRepository) UpdateBilling(ctx context.Context, billingId string, billingDetail entity.BillingDetail) (*entity.BillingDetail, error) {
+	billing := b.db.Database("billing").Collection("billings")
+
+	id, err := primitive.ObjectIDFromHex(billingId)
+	if err != nil {
+		return nil, utils.ErrorInvalidPrimitiveID
+	}
+	chargedMemberIdConvert, err := primitive.ObjectIDFromHex(billingDetail.ChargedMemberId)
+	if err != nil {
+		return nil, utils.ErrorInvalidPrimitiveID
+	}
+
+	filter := bson.D{{"_id", id}}
+	dataUpdate := bson.D{{"$set", bson.D{
+		{"title", billingDetail.Title},
+		{"billAmount", billingDetail.BillAmount},
+		{"chargedMemberId", chargedMemberIdConvert},
+		{"isBillEqually", billingDetail.IsBillEqually},
```

Comment:
ini kayaknya perlu dijadiin snake case ya ? jadinya gini ? 

```suggestion
		{"title", billingDetail.Title},
		{"bill_amount", billingDetail.BillAmount},
		{"member_id", chargedMemberIdConvert},
		{"is_bill_equal", billingDetail.IsBillEqually},
```

cmiiw @debbysa @milhamsuryapratama

Reply:
iyes

Reply:
jadinya aku pakek `is_bill_equally` disini. di createBilling udah ku update juga